src/app/cube/shared/learning-object/learning-object.component.html

```diff
@@ -51,9 +51,8 @@
       </div>
       <div class="details">
         <span class="author">
-          {{ learningObject.author.name | titlecase }} |
-          {{ learningObject.author.organization.trim() | titlecase
-          }}{{
+          {{learningObject.contributors[0]?.name | titlecase}}
+          {{
             learningObject.contributors?.length
               ? ', and ' + learningObject.contributors.length + ' more'
```

Comment:
Subtract one from the length so it is accurate. You're displaying the first one.